### PR TITLE
Rewards

### PR DESCRIPTION
**Pull request by cpfergus1**
## Description
Implemented Rewards table which belongs to both the user and the parent. This was not name-spaced on either model due to it having multiple associations. 

## How Has This Been Tested?

96% tested  for rewards - will return to finish testing

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have updated the README to reflect any changes.

### CODE REVIEW

**Comment by cpfergus1** [comment]
Remember to get on heroku and run rails db:migrate db:seed after the merge

File: app/models/parent.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Parent < ApplicationRecord
+  has_many :rewards
```
Comment (dunlapww) [inline app/models/parent.rb]: Think we want to add a 'dependent: destroy' after the has_many :rewards

**Review by dunlapww — APPROVED** [review]
Look great Connor, thanks for knocking this out!

File: app/models/parent.rb (hunk 1 above)
Comment (cpfergus1) [reply app/models/parent.rb]: updated with latest commit!

**Pull request merged**